src/jit/lsra.cpp

```diff
@@ -10355,6 +10376,23 @@ void TreeNodeInfo::Initialize(LinearScan* lsra, GenTree* node, LsraLocation loca
     else
     {
         dstCandidates = genRegMask(node->gtRegNum);
+#ifdef ARM_SOFTFP
+        regNumber reg2 = REG_NA;
+        if (node->OperGet() == GT_PUTARG_REG)
```

Comment:
Yes I think so. However there was no code for `GT_MUL_LONG` here so I'm not sure if I should add that. I'm not sure if there is a case that GT_MUL_LONG node is actually generated. So I left it out.